src/MoBi.Core/Domain/UnitSystem/MoBiDimensionFactory.cs

```diff
@@ -8,14 +8,14 @@ namespace MoBi.Core.Domain.UnitSystem
 {
    public interface IMoBiDimensionFactory : IDimensionFactory
    {
-      IDimension TryGetDimensionCaseInsensitive(string dimensionName);
```

Comment:
The old signature threw me a curve ball. In the implementation at least the parameter is named appropriately